prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseConfiguration.cs

```diff
@@ -103,6 +103,23 @@ public override IEnumerable<License> SeedData
                     new License { Code = 85, Weight = 4, Name = "Non-practicing registration" },
                     new License { Code = 86, Weight = 4, Name = "Temporary registration" },
 
+                    //College of Oral Health Professionals
+                    new License { Code = 87, Weight = 1, Name = "Dental Hygienists" },
+                    new License { Code = 88, Weight = 2, Name = "Dental Hygiene Practitioner" },
+                    new License { Code = 89, Weight = 3, Name = "Full Dentist" },
+                    new License { Code = 90, Weight = 4, Name = "Limited Dentist (Restricted to Specialty)" },
+                    new License { Code = 91, Weight = 5, Name = "Limited (Academic) Dentist" },
+                    new License { Code = 92, Weight = 6, Name = "Limited (Educational and Volunteer) Dentist" },
+                    new License { Code = 93, Weight = 7, Name = "Limited (Armed Forces/Government) Dentist" },
+                    new License { Code = 94, Weight = 8, Name = "Dental Therapist" },
+                    new License { Code = 95, Weight = 9, Name = "Dental Technician" },
+                    new License { Code = 96, Weight = 10, Name = "Full Denturist" },
+                    new License { Code = 97, Weight = 11, Name = "Student Dentistry" },
+                    new License { Code = 98, Weight = 12, Name = "Limited (Grandparented) Denturist" },
+                    new License { Code = 99, Weight = 13, Name = "Student dental technology" },
+                    new License { Code = 100, Weight = 14, Name = "Student Denturism" },
+                    new License { Code = 101, Weight = 15, Name = "Temporary" },
+
```

Comment:
Please order as per the list in https://bcgovmoh.atlassian.net/browse/PRIME-2320?focusedCommentId=15098

Reply:
total forgot about it. it is fixed.